darts/models/torch_forecasting_model.py

```diff
@@ -327,19 +362,17 @@ def fit_from_dataset(self,
                          verbose: bool = False,
                          epochs: int = 0) -> None:
```

Comment:
The docstring for `epochs` in `fit()` says
```
If your model is already trained but you want to train it even further, you can provide here the number of additional epochs.
```
I would change it to
```
If specified, will train the model for `epochs` (additional) epochs, irrespective of what `n_epochs` was provided to the model constructor.
```

Reply:
But we don't want it at all, do we? (based on the discussion above)

Reply:
The conversation above was about how to use `force` and `reset_model()` when instantiating a model with an existing name. However once a given model is instantiated, it still makes sense to accept the `epochs` parameter in the `fit()` function in order to potentially override the `n_epochs` class parameter, no? I think it's a nice feature :)

Reply:
oh, sorry - I didn't notice that it's about the docstring...